codechain/crypto/bech32.py

```diff
@@ -0,0 +1,109 @@
+CHARSET = "qpzry9x8gf2tvdw0s3jn54khce6mua7l"
```

Comment:
I think you've borrowed code from here: https://github.com/sipa/bech32/blob/master/ref/python/segwit_addr.py
There's a copyright notice, and I think it is a MIT license. It says:
```
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
```
And you should follow it. If you modified the code, then here's how you should do it:
https://softwareengineering.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project

Reply:
I added the copyright claim.